Source/Scene/ModelExperimental/CustomShaderGuide/README.md

```diff
@@ -227,34 +216,57 @@ This struct is similar to `VertexInput`, but there are a few more automatic
+| `TANGENT`                        | `tangentMC`        | `vec3`  | Yes                         | No                            | Unit-length tangent vector in model coordinates. This is always a `vec3`. For models that provide a `w` component, that is removed after computing the bitangent vector. |
+| `TANGENT`                        | `tangentEC`        | `vec3`  | No                          | Yes                           | Unit-length tangent vector in eye coordinates. This is always a `vec3`. For models that provide a `w` component, that is removed after computing the bitangent vector.   |
+| `NORMAL` & `TANGENT`             | `bitangentMC`      | `vec3`  | Yes                         | No                            | Unit-length bitangent vector in model coordinates. Only available when both normal and tangent vectors are available.                                                    |
+| `NORMAL` & `TANGENT`             | `bitangentEC`      | `vec3`  | No                          | Yes                           | Unit-length bitangent vector in eye coordinates. Only available when both normal and tangent vectors are available.                                                      |
+| `TEXCOORD_N`                     | `texCoord_N`       | `vec2`  | Yes                         | Yes                           | `N`-th set of texture coordinates.                                                                                                                                       |
+| `COLOR_N`                        | `color_N`          | `vec4`  | Yes                         | Yes                           | `N`-th set of vertex colors. This is always a `vec4`; if the model does not specify an alpha value, it is assumed to be 1.                                               |
+| `JOINTS_N`                       | `joints_N`         | `ivec4` | Yes                         | Yes                           | `N`-th set of joint indices                                                                                                                                              |
+| `WEIGHTS_N`                      | `weights_N`        | `vec4`  |
+
+Custom attributes are also available, though they are renamed to use lowercase
+letters and underscores. For example, an attribute called `_SURFACE_TEMPERATURE`
+in the model would become `fsInput.attributes.surface_temperature` in the shader.
+
 ## `czm_modelMaterial` struct
 
-This one is a built-in, see the [documentation comment](https://github.com/CesiumGS/cesium/blob/model-experimental-custom-shaders/Source/Shaders/Builtin/Structs/modelMaterial.glsl). This is similar to `czm_material` from the old Fabric system, but slightly different fields as this one supports PBR lighting.
+https://github.com/CesiumGS/cesium/tree/main/Source/Scene/ModelExperimental/CustomShaderGuide
+../../../
```

Comment:
Is this link correct?

Reply:
whoops this shouldn't be here, I think I had this line here for reference to make sure I had the right number of `../` but then forgot to remove it :sweat_smile: